www/md_base/src/icons/gear-outline.svg

```diff
@@ -0,0 +1,40 @@
+<?xml version="1.0" encoding="utf-8"?>

```

Comment:
We are using `md-icon`, which is a wonderful solutions for using SVG icons. All SVG files will be merged into a single file and Angular Material will cache it with `templateCache`.

The using of these icons is nearly the same as using an icon font. `md-icon` will handle everything and we no longer need to include at least 3 font files (.svg, .woff, .ttf) as well as the icons that are not used.

The compatibility is not a problem for our project. Every browsers that supports AngularJS have already support SVG.

So I insist using SVG icons.